packages/driver/cypress/e2e/commands/files.cy.js

```diff
@@ -176,10 +178,6 @@ describe('src/cy/commands/files', () => {
 
         this.logs = []
 
-        cy.on('fail', () => {
```

Comment:
Having an `on('fail')` handler in the `beforeEach()` block makes it hard to debug actual failures, since if you comment out a test's onFail block, the test will suddenly pass.

The only change this makes is that in some tests we see three logs rather than two, because "the stub has been called" log is included (where previously it would have been ignored because of this handler-removal).